refinery/ui/source/js/home/directives/home.spec.js

```diff
@@ -22,7 +22,7 @@
       $httpBackend
         .expectGET(
           settings.appRoot +
-          settings.refineryApiV2 + '/site_profiles/'
+          settings.refineryApiV2 + '/site_profiles/?current_site=true'
```

Comment:
For params being sent, it appears we typically sent it to default to django's 'True'.